opentaskspal/src/main/java/org/dmfs/opentaskspal/rowsets/Subtasks.java

```diff
@@ -35,10 +37,15 @@
  */
 public final class Subtasks extends DelegatingRowSet<TaskContract.Tasks>
 {
+    public Subtasks(@NonNull String authority, @NonNull ContentProviderClient client, @NonNull Uri parentTask, String... projection)
+    {
+        this(new TasksView(authority, client, projection), parentTask);
+    }
+
 
-    public Subtasks(String authority, ContentProviderClient client, @NonNull RowSnapshot<TaskContract.Tasks> parentTask)
+    public Subtasks(View<TaskContract.Tasks> view, @NonNull Uri parentTask)
```

Comment:
This ctor should take a `RowReference`, everything else (`Uri`, `RowSnapshot` ) is subject to secondary ctors.

Reply:
But we need `authority` and `client` and `projection` as well, right? So:
```java
public Subtasks(@NonNull RowReference<Tasks> parentTask, @NonNull String authority, @NonNull ContentProviderClient client, @NonNull String... projection)
```
(I will probably create separate ticket for these taskspal related changes as well, later.)

Reply:
the primary ctor should only take a `View<Tasks>` and the `RowReference<Tasks>`.
Everything else (including creation of a `View<Tasks>` from `authority`, `client` and `projection`) is merely for convenience and should be done by secondary ctors.

Reply:
Ok, I've update the constructor.